cypress/e2e/shared/explorer.test.ts

```diff
@@ -569,6 +569,83 @@ describe('DataExplorer', () => {
         .should('exist')
         .should('contain', taskName)
     })
+
+    it('injects variables into flux-editor', () => {
+      const fluxQuery: Array<string> = [
+        'from(bucket: "defbuck")',
+        '  |>range(start: , stop: )',
+      ]
+
+      cy.get('.view-line')
+        .last()
+        .type(fluxQuery.join('{enter}'))
```

Comment:
this is fine but just wanted to mention it.. just in case. you can also use..

```
`${fluxQuery}{enter}`
```

Reply:
I'll take note of that, thanks.